auth/client/src/main/resources/schema/elytron-client-1_7.xsd

```diff
@@ -97,7 +97,8 @@
                 </xsd:annotation>
             </xsd:element>
             <xsd:element name="providers" type="providers-type" minOccurs="0"/>
-            <xsd:element name="certificate-revocation-list" type="certificate-revocation-list-type" minOccurs="0"/>
+            <xsd:element name="certificate-revocation-list" type="certificate-revocation-list-type" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="certificate-revocation-lists" type="certificate-revocation-lists-type" minOccurs="0" maxOccurs="1" />
```

Comment:
Just FYI, maxOccurs defaults to 1 so it can be left out here.

Reply:
okay, I removed it :)